js/script.js

```diff
@@ -1,3 +1,5 @@
+$(document).ready(function () {
+
 $('#choose').on('change', function(){
```

Comment:
With the addition of the document.ready function, fix the indentation through out the whole file.